src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -606,6 +606,12 @@ void mongoSubCountersUpdate
 {
   std::string  collection = getSubscribeContextCollectionName(tenant);
 
+  if (subId == "")
+  {
+    // LM_E(("Runtime Error (no subscription id)"));
```

Comment:
Discussed at skype.

This situation should be detected previously to call this function, i.e. in the startSenderThread() function, based in a new field in SenderThreadParams to identify the origin (either NGSI10 or NGSI9) of the notification.